### PR TITLE
Add tests extract_latlon_dataarray

### PR DESCRIPTION
**Pull request by ejgal**
The old test only checked that `extract_latlon_dataarray` returned a dataarray.
Added a few test cases.

### CODE REVIEW

File: pyaerocom/test/test_helpers.py [hunk 1]
```diff
@@ -125,7 +126,17 @@ def test_extract_latlon_dataarray():
...
+    assert len(subset.lat) == len(lat) - 1 and len(subset.lon) == len(lon) -1
 
+@pytest.mark.parametrize('lat,lon,exception', [
+    ([], [], DataCoverageError),
+    ([1,2], [-1,2], DataCoverageError)
+    ])
+def test_extract_latlon_dataarray_no_matches(lat, lon, exception):
+    cube = helpers.make_dummy_cube_latlon(lat_res_deg=1, lon_res_deg=1, lat_range=[10,20], lon_range=[10,20])
+    data = xr.DataArray.from_iris(cube)
+    with pytest.raises(DataCoverageError):
```
Comment (jgliss) [inline pyaerocom/test/test_helpers.py:138]: You do not use input `exception`. Shouldn't it be `pytest.raises(exception)`?

**Comment by ejgal** [comment]
Thanks, that way seems great, looking into it.

File: pyaerocom/test/test_helpers.py (hunk 1 above)
Comment (ejgal) [reply pyaerocom/test/test_helpers.py:138]: Yes!

**Comment by ejgal** [comment]
@jgliss, updated this now, can you have a quick look at it?

File: pyaerocom/test/test_helpers.py [hunk 2]
```diff
@@ -125,7 +128,22 @@ def test_extract_latlon_dataarray():
     lon = [1,15, 18]
     subset = helpers.extract_latlon_dataarray(data, lat, lon, check_domain=True)
     assert isinstance(subset, xr.DataArray)
-
+    assert len(subset.lat) == len(lat) - 1 and len(subset.lon) == len(lon) -1
+
+@contextmanager
+def does_not_raise_exception():
+    yield
```
[the remark this answers is not in the document]
Comment (jgliss) [reply pyaerocom/test/test_helpers.py:135]: @ejgal still waiting for this :)

[the remark this answers is not in the document]
Comment (ejgal) [reply pyaerocom/test/test_helpers.py:135]: Almost, you added it in pyaerocom/**web**/test/test_helpers.py, this is pyaerocom/test/test_helpers.py.
Maybe `does_not_raise_exception` would be nice to have in conftest instead?

[the remark this answers is not in the document]
Comment (jgliss) [reply pyaerocom/test/test_helpers.py:135]: Absolutely! Go ahead :)

[the remark this answers is not in the document]
Comment (ejgal) [reply pyaerocom/test/test_helpers.py]: Done.

**Pull request merged**